Abcbadq/sjsu_shop
# Cart in history

Minh-What files did I changed:
+ CheckoutCart.js
+ api/profile.js
+ models/Profile.js
 => to add the ability to add carts in user's profile history.
Testing:
+ click pay button should add carts object in profile history
+ go to MLab to check if the carts are there

Kento-What files did I changed:
+ CheckoutCart.js
+ Delivery.js
+ cartActions.js
Testing:
+ add item to cart, checkout, add delivery details (the form doesn't have checks yet)
+ log in with profile setup, add item to cart, checkout Both pages should end up at /recipt page

Yu Ning-What files did I change;
+ CheckoutCart.js
+ ShoppingCart.js
+ cartActions.js
+ cartReducer.js
+ types.js
Testing: 
+ input 15OFF or BlackFriday as discount codes, then click Apply Discount button
+ Displays correct discount percentage (15%, 10% respectively) and the correct price after discount in both shopping cart and checkout.
